app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivityTest.java

```diff
@@ -170,4 +179,17 @@ public void whenAllNecessaryPermissionsAreGranted_prepareViewIsCalled() {
         //Then
         verify(spyActivity).prepareView();
     }
+
+    private EditActivity createActivity() {
```

Comment:
I need this only for 3 tests so in my opinion its redundant, however i don't see problem with put this is setup method